client/landing/stepper/declarative-flow/internals/steps-repository/import-list/index.tsx

```diff
@@ -1,25 +1,29 @@
-import { type FC } from 'react';
 import ListStep from 'calypso/blocks/import/list';
-import { type StepProps } from 'calypso/landing/stepper/declarative-flow/internals/types';
+import { type Step } from 'calypso/landing/stepper/declarative-flow/internals/types';
 import { useSiteSlug } from 'calypso/landing/stepper/hooks/use-site-slug';
+import { ImporterPlatform } from 'calypso/lib/importer/types';
```

Comment:
Nit: this looks like it should be a type import.
```suggestion
import type { ImporterPlatform } from 'calypso/lib/importer/types';
```

Reply:
Fixed.